src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size.mdx

```diff
@@ -26,7 +26,11 @@ Agent version 5.0.0 or higher.
 
 ## Description
 
-By default, mobile monitoring collects a maximum of 1,000 events per [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle) (cycles are ten minutes long by default). This method lets you override the maximum size of that event pool. When the pool size limit is reached, the New Relic Android agent will begin [sampling events](/docs/agents/manage-apm-agents/agent-data/new-relic-events-limits-sampling), discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle.
+By default, mobile monitoring collects a maximum of 1,000 events per [harvest cycle](/docs/apm/new-relic-apm/getting-started/glossary#harvest-cycle) (cycles are ten minutes long by default). 
```

Comment:
Per _event harvest_ cycle, which is 600 secs (default). The normal harvest cycle is 60 sec (fixed). Events are also harvested when the app goes to background.